elephant/sta.py

```diff
@@ -314,3 +320,57 @@ def spike_field_coherence(signal, spiketrain, **kwargs):
 
     return (pq.Quantity(sfc, units=pq.dimensionless),
             pq.Quantity(frequencies, units=pq.Hz))
+
+
+def stPR(spiketrains, bin_size=1*pq.ms, sigma=5*pq.ms):
+    """
+    Spike triggered population response as described in [1].
+
+    Parameters
+    ----------
+    spiketrains : list of neo.SpikeTrains
+        list of spike train objects containing a list of spiketimes. The spike
```

Comment:
we never indicate what a spiketrain is
I'd not put `containing a list of spiketimes` because one might think that `[1*pq.ms, 9*pq.ms]` is a valid spiketrain object, according to your definition, while it's not.

Reply:
fixed